app/main.py

```diff
@@ -1,6 +1,16 @@
+import os
+import requests
+
+
+API_KEY = os.environ.get("API_KEY")
+URL = "https://api.weatherapi.com/v1/current.json"
+CITY = "Paris"
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    params = {"key": API_KEY, "q": CITY}
+    res = requests.get(url=URL, params=params).json()
+    print(res)
```

Comment:
You should check http code of response. If code is not 200, you should show some notification about it to user